### PR TITLE
Set local replica address to podIP:0

### PR DESCRIPTION
**Pull request by derekbit**

### CODE REVIEW

File: pkg/spdk/engine.go [hunk 1]
```diff
@@ -205,7 +205,7 @@ func svcEngineGet(spdkClient *spdkclient.Client, name string) (res *spdkrpc.Engi
 		if !exists {
 			// This replica must be a local lvol
 			replicaName := spdktypes.GetLvolNameFromAlias(baseBdev.Name)
-			res.ReplicaAddressMap[replicaName] = ""
+			res.ReplicaAddressMap[replicaName] = fmt.Sprintf("%s:0", podIP)
```
Comment (shuo-wu) [inline pkg/spdk/engine.go:208]: NIT: use `net.JoinHostPort()`

Comment (derekbit) [reply pkg/spdk/engine.go:208]: One question:
Does local replica still allocate a port? @shuo-wu

Comment (derekbit) [reply pkg/spdk/engine.go]: Updated. Thank you.

**Comment by derekbit** [comment]
No need after introducing engine cache.

**Pull request closed without merging**